### ISSUE TITLE
Ethernet and PC ?

### ISSUE THREAD

**Issue by tuxcnc** [iforce2d/weenyPRU#3]
There is https://github.com/multigcs/LinuxCNC-RIO project.
It was intended for Raspberry and SPI transport too, but there is possibility to use PC and Ethernet to SPI converter module (eg. WT32-ETH01) and UDP transport.
I suspect, there is possibility to write some code for this project, and use STM32 driver with PC computer...

**Comment by iforce2d** [comment]
Yes, it would be nice not to need a specific single-board computer to run this. For now I'm struggling to keep up with developing and maintaining just this target, I barely learned how to get it done with Raspberry Pi. The ethernet-SPI bridge looks like a huge mountain of study and research required...
